service/routes.py

```diff
@@ -306,6 +306,56 @@ def update_wishlist_items(wishlist_id, item_id):
     )
 
 
+######################################################################
```

Comment:
This is a duplicate of the route handler on 198.  We should just be adding the parameter handling to the existing function.  This has dropped the code coverage down because none of these lines are being run.

Reply:
Hi Jay, I modified the list function to support the query feature. I pushed the changes and it passed the code coverage test. Thanks for your advice!